dags/walg_backups.py

```diff
@@ -0,0 +1,28 @@
+from airflow.operators.python_operator import PythonOperator
+from datetime import datetime, timedelta
+from exec_in_pod import exec_in_pod
+import os
+
+START_DATE = datetime.now() - timedelta(days=2)
+
+ciip_namespace = os.getenv('CIIP_NAMESPACE')
+ggircs_namespace = os.getenv('GGIRCS_NAMESPACE')
+in_cluster = os.getenv('LOCAL_AIRFLOW', False) == 'False'
```

Comment:
doesn't look like this is used... which is a good thing because it's bugged :smile: , it should be 
```py
os.getenv('LOCAL_AIRFLOW', 'False') == 'False'
```

Reply:
the other variables above, and the `datetime` import don't seem to be used either

Reply:
good catch, I copy-pasted just a little too fast